js/my.js

```diff
@@ -232,3 +237,30 @@ document.addEventListener("DOMContentLoaded", function () {
   });
 });
 
+function parseCommand(client, nick, host, cmd) {
+  cmd = cmd.replace(/^\//, '');
```

Comment:
The `replace` is doing more work than is necessary; the regex two lines down should work either way.